src/telemetry/metrics.ts

```diff
@@ -10,26 +10,30 @@ import { GcpDetectorSync } from '@google-cloud/opentelemetry-resource-util';
 import { isProd } from '../common';
 
 export const startMetrics = (serviceName: string): void => {
-  const metricReader = isProd
-    ? new metrics.PeriodicExportingMetricReader({
+  const readers: metrics.MetricReader[] = [
+    new PrometheusExporter({}, () => {
+      const { endpoint, port } = PrometheusExporter.DEFAULT_OPTIONS;
+      console.log(`metrics endpoint: http://localhost:${port}${endpoint}`);
```

Comment:
Use the logger? Or do we need it even?

Reply:
I removed the console.log, and added logger for errors instead.